src/popup/components/MultisigVaultCreateReview.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+  <div class="multisig-vault-create-review">
+    <h2 class="text-heading-1">
+      {{ $t('modals.createMultisigAccount.title') }}
+    </h2>
+
+    <DetailsItem :label="$t('multisig.creatingAccount')">
+      <template #value>
+        <div class="creator">
+          <Avatar :address="creatorAccount.address" />
+          <div class="creator-data">
+            <BtnPill
+              class="account-select"
+              dense
+            >
+              <FormSelect
+                v-model="creatorAddress"
+                unstyled
+                :default-text="$t('modals.createMultisigAccount.selectAccount')"
+                :options="accountsOptions"
+              />
+            </BtnPill>
+            <AddressTruncated
+              show-explorer-link
+              :address="creatorAccount.address"
+            />
+          </div>
+        </div>
+      </template>
+    </DetailsItem>
+
+    <div class="review-details-row">
+      <DetailsItem :label="$t('multisig.authorizedSigners')">
+        <template #value>
+          <div class="authorized-signers">
+            <div
+              v-for="signer in signers"
+              :key="signer.address"
+              class="authorized-signers-row"
+            >
+              <AccountItem :address="signer.address" />
+              <DialogBox
+                v-if="isLocalAccountAddress(signer.address)"
+                dense
+                v-text="$t('common.you')"
+              />
+            </div>
+          </div>
+        </template>
+      </DetailsItem>
+
+      <DetailsItem :label="$t('multisig.consensus')">
+        {{ confirmationsRequired }}/{{ confirmationsRequired }} {{ $t('common.of') }} {{ signers.length }}
```

Comment:
that string you can get from composables. Props name is "consensulLabel" You can check multisigDetails.vue

Reply:
Unfortunately not in this case because this is a new account, not a one that was retrieved. And here you can see that both numbers before and after the slash are the same.